x/mongo/driver/topology/topology.go

```diff
@@ -142,15 +143,20 @@ func New(opts ...Option) (*Topology, error) {
 		return t.apply(context.TODO(), desc)
 	}
 
-	if t.cfg.uri != "" {
-		t.pollingRequired = strings.HasPrefix(t.cfg.uri, "mongodb+srv://") && !t.cfg.loadBalanced
+	if t.cfg.URI != "" {
+		t.pollingRequired = strings.HasPrefix(t.cfg.URI, "mongodb+srv://") && !t.cfg.LoadBalanced
 	}
 
 	t.publishTopologyOpeningEvent()
 
 	return t, nil
 }
 
+// NewDefault creates a new unconfigured topolgoy.
+func NewDefault() (*Topology, error) {
```

Comment:
This could be dropped in favor of using `New(nil)` inline, though this does look a little strange.